src/components/Tooltip/index.tsx

```diff
@@ -1,51 +1,40 @@
 import React from 'react';
 import FDS from 'dictionary/js/styleConstants';
-import Popover, { Position } from 'components/Popover';
-
-export const DEFAULT_WIDTH = '240px';
-export const DELAY_MS = 350;
-export const DEFAULT_POSITION = 'bottom';
+import * as RadixTooltip from '@radix-ui/react-tooltip';
 
 export interface TooltipProps {
-  /** JSX - Acts as a positioning reference for the popover and triggers active state */
+  /** What we're showing tooltip on */
   trigger: JSX.Element | string;
 
-  /** Content of the tooltip */
+  /** The actual content of the tooltip */
   message: string;
 
   /** Maximum width of tooltip */
   maxWidth?: number;
 
-  /**
-   * Position preference of tooltip.
-   * `top` for example, will place the tooltip above the trigger.
-   */
-  position?: Position;
+  /** Position preference of tooltip. */
+  position?: 'top' | 'left' | 'right' | 'bottom';
```

Comment:
`position` is where the tooltip content shows **related** to `trigger`.

example:
`position = 'left'`, it will show content to the left of trigger.